cmake/CeleritasAddTest.cmake

```diff
@@ -338,7 +338,7 @@ function(celeritas_add_test SOURCE_FILE)
   endif()
 
   set(_COMMON_PROPS)
-  if(CELERITAS_USE_CUDA)
+  if(CELERITAS_USE_DEVICE)
```

Comment:
I think this is the only use of this variable.  Everywhere else you do
```cmake
(CELERITAS_USE_CUDA or CELERITAS_USE_HIP)
```
My guess is this is always false or I'm missing something (very possible).

Reply:
ooh good catch, thanks! I think I'd defined it at some point but it got refactored out.